Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -8025,6 +8025,8 @@ void WebPageProxy::decidePolicyForResponseShared(Ref<WebProcessProxy>&& process,
     MESSAGE_CHECK_URL_COMPLETION(process, response.url(), completionHandler({ }));
     RefPtr navigation = navigationID ? protectedNavigationState()->navigation(*navigationID) : nullptr;
     Ref navigationResponse = API::NavigationResponse::create(API::FrameInfo::create(WTFMove(frameInfo), this).get(), request, response, canShowMIMEType, downloadAttribute);
+    if (wasPrivateRelayed)
+        navigationResponse->setWasPrivateRelayed();
```

Comment:
This value should go into the constructor.

Reply:
done